src/smt.rs

```diff
@@ -473,6 +497,10 @@ impl QueryProofWithProof {
         self.binary_bitmap.len()
     }
 
+    fn is_empty_height(&self) -> bool {
```

Comment:
what does it mean by `empty height`?

Reply:
The above function returns `binary_bitmap` len as `height` so I made it to check the `height` is empty or no.

Reply:
I think height `0` means root, so `is_root` or `is_zero_height` would be better?

Reply:
rename to `is_zero_height`